### DISCUSSION TITLE
Suspicious exe file

### DISCUSSION THREAD

**Discussion by davidovic999** [leokhoa/laragon#557] [General]
Hi,

I found your application very interesting compared to others.
But your executable contains trojans and malware and please don't tell me they're false positives!
How can I trust you? It's really confusing.

**Comment by leokhoa** [comment]
@davidovic999 : 
I suggest the following steps:

1. Verification with the AV Vendor: Please help to reach out to the AV vendor to confirm the validity of the detected issues. Laragon has a feature that need to modify hosts file (Auto virtual host) an add environment variables to system path which may confuse them.

2. Collaboration and Improvement: If the reported issues are indeed false positives, I would greatly appreciate your assistance in sharing the results of your verification process with us. This will not only help me rectify any misunderstandings but also contribute to improving Laragon's security measures. Otherwise, I want to know the root cause so I fix can the issue to make users feel safe.

3. Independent Verification: Furthermore, you could consider seeking a third-party cybersecurity expert to conduct an independent assessment of Laragon executable. I am confident that such an evaluation will help clarify the situation and reinforce your trust in the software.

**Comment by 14okt** [comment]
apparently, this is a problem with the "Mail Catcher" feature code being detected as spyware..

**Comment by leokhoa** [comment]
thanks @14okt : I will remove this feature as it needs to catch the output from PHP mail which cause this alarm.

**Reply by RobiNN1** [reply]
Replace it with https://github.com/axllent/mailpit

**Comment by 14okt** [comment]
it's better not to delete it, but change the code, use an obfuscator like this https://github.com/andrivet/ADVobfuscator

**Comment by Skintillion** [comment]
@davidovic999 It's a false positive
